webapp/src/components/modals/full_screen_modal/full_screen_modal.scss

```diff
@@ -0,0 +1,43 @@
+.FullScreenModal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: var(--center-channel-bg);
+    z-index: 2000;
+    overflow: auto;
+
+    .close-x {
+        position: fixed;
+        top: 48px;
+        right: 48px;
+        cursor: pointer;
+        svg {
+            fill: var(--center-channel-color-90);
+            padding: 4px;
+            width: 40px;
+            height: 40px;
+            border-radius: 4px;
+            &:hover {
+                background: var(--button-bg-10);
+            }
+        }
+    }
+
+    // Animation
+    &-enter, &-appear {
+        opacity: 0;
+    }
+    &-enter-active, &-appear-active {
+        opacity: 1;
+        transition: opacity 100ms;
```

Comment:
Can we make the animation duration a var, then it's easier to sync the two files?